app/controllers/home_controller.rb

```diff
@@ -1,3 +1,5 @@
 class HomeController < ApplicationController
-  def index; end
+  def index
+    redirect_to account_root_path if user_signed_in?
```

Comment:
remove this line. what if I want to visit home page.

Reply:
Removed.